audio.py

```diff
@@ -88,3 +137,57 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+class AudioTests(unittest.TestCase):
```

Comment:
Let's get rid of these tests as we discussed on the call (one reason is that our test data aren't committed in the repo, another reason is that this code won't change frequently either way; in the long run tests would be good, but we should discuss this separately).

As we discussed on the call, please replace these with a `if __name__ == '__main__'` section that accepts arguments from command line and prints average measurements. Currently that section does not print any measurements